tools/integration-tests/tester/tests/consensus/consensus_conflict_spam_test.go

```diff
@@ -0,0 +1,185 @@
+package consensus
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/iotaledger/hive.go/crypto/ed25519"
+
+	"github.com/iotaledger/goshimmer/packages/consensus/gof"
+	"github.com/iotaledger/goshimmer/packages/jsonmodels"
+	"github.com/iotaledger/goshimmer/packages/ledgerstate"
+	"github.com/iotaledger/goshimmer/tools/integration-tests/tester/framework"
+	"github.com/iotaledger/goshimmer/tools/integration-tests/tester/tests"
+	"github.com/stretchr/testify/require"
+)
+
+// constant var, shouldn't be changed
+var tokensPerRequest int
+
+/**
+TestConflictSpam spams a node with conflicts and makes sure the GoFs are the same across the network
+*/
+func TestConflictSpam(t *testing.T) {
+	ctx, cancel := tests.Context(context.Background(), t)
+	defer cancel()
+	n, err := f.CreateNetwork(ctx, t.Name(), 4, framework.CreateNetworkConfig{
+		Faucet:      true,
+		StartSynced: true,
+		Activity:    true,
+	}, tests.EqualDefaultConfigFunc(t, false))
+	require.NoError(t, err)
+	defer tests.ShutdownNetwork(ctx, t, n)
+
+	faucet, peer1 := n.Peers()[0], n.Peers()[1]
+	tokensPerRequest = faucet.Config().TokensPerRequest
+
+	tests.AwaitInitialFaucetOutputsPrepared(t, faucet, n.Peers())
+
+	fundingAddress := peer1.Address(0)
+	tests.SendFaucetRequest(t, peer1, fundingAddress)
+	require.Eventually(t, func() bool {
+		return tests.Balance(t, peer1, fundingAddress, ledgerstate.ColorIOTA) >= uint64(tokensPerRequest)
+	}, tests.Timeout, tests.Tick)
+
+	addresses := make([]*ledgerstate.Address, 100)
+	keyPairs := map[string]*ed25519.KeyPair{}
+	for i := 0; i < 100; i++ {
+		address := peer1.Address(i)
+		addresses[i] = &address
+		keyPairs[address.String()] = peer1.KeyPair(uint64(i))
+	}
+
+	outputs := getOutputsControlledBy(t, peer1, fundingAddress)
+	fundingKeyPair := map[string]*ed25519.KeyPair{fundingAddress.String(): peer1.KeyPair(0)}
+	outputs = splitToAddresses(t, peer1, outputs[0], fundingKeyPair, addresses...)
+	const conflictRepetitions = 4
+	pairwiseOutputs := outputs[:(conflictRepetitions-1)*3+3]
+	tripletOutputs := outputs[len(pairwiseOutputs):]
+
+	txs := []*ledgerstate.Transaction{}
```

Comment:
Use a `make`

Reply:
I changed to `make` in several places. Here I would rather not.
If I understand correctly you want to use `make` in order to create an array of certain predefined size and use indexes. This in order to decrease undelrying `malloc` operations.

But in this case, it will make the test harder to change. Because the size of txs slice may change if we add more conflict types or do other changes.

...Or maybe you wanted me to use `make` to create a 0 size slice? But then what is the advantage of using this syntax? Is it just a style thing that we have that I don't know about?

Reply:
I just find `make` more go-like: not really saying this because of malloc optimizations :smile:

Reply:
Why is it more go style in your opinion? I am actually curious because never saw anything in the docs about it.

Anyhow, I don't mind using only `make` for empty slice allocation, so I am changing it. I just wish we had a style guide so I wouldn't have to guess this stuff. Back in the IRI days I just shamelessly copied the style guide from another project that I liked.
Maybe we can do the same?
I can also learn the style via comments though if it is too much :-P